include/ensmallen_bits/ens_version.hpp

```diff
@@ -37,7 +41,19 @@ struct version
 
     std::stringstream ss;
     ss << version::major << '.' << version::minor << '.' << version::patch
-       << " (" << nickname << ')';
+       << " (" << nickname << ")";
+
+    return ss.str();
+  }
+    
+  static inline std::string date()
+  {
+    const char* year = ENS_VERSION_YEAR;
+    const char* month = ENS_VERSION_MONTH;
+    const char* day = ENS_VERSION_DAY;
+
+    std::stringstream ss;
+    ss << year << '-' << month << '-' << day;
```

Comment:
What is the reason to not pass the date into the stream directly:

`ss << ENS_VERSION_YEAR << '-' << ENS_VERSION_MONTH << '-' << ENS_VERSION_DAY;`

Reply:
@zoq No reason not to. I opted to mirror the styling of `nickname` if memory serves correctly.